apps/web/pages/[user].tsx

```diff
@@ -310,6 +329,7 @@ export const getServerSideProps: GetServerSideProps<UserPageProps> = async (cont
   const safeBio = markdownToSafeHTML(user.bio) || "";
 
   const markdownStrippedBio = stripMarkdown(user?.bio || "");
+  const unPublishedOrgUser = usersWithoutAvatar.find((us) => us.organization?.slug === null);
```

Comment:
nit:
I think if we can get the org directly. All users would have the same org because of the above query.
`const org = usersWithoutAvatar[0].organization`

and the modify the below code like this
`isUnpublished: org.slug == null`
`name: org.name`


It would IMHO, make the code more readable.

Reply:
Good call, made the change, thanks!